src/components/wish/WishDetail.jsx

```diff
@@ -0,0 +1,108 @@
+import styled from 'styled-components';
+import HeartButton from 'components/common/HeartButton';
+import { client } from 'libs/api';
+
+function WishDetail({ detail, setWishDetail }) {
+  const toggleHeart = async (id, currentHeartStatus) => {
+    await client.patch(`/wish/${id}`, {
+      like: !currentHeartStatus,
+    });
+
+    const newDetail = detail.map((post) => {
+      if (post.id === id) {
+        return {
+          ...post,
+          like: !currentHeartStatus,
+        };
+      }
+
+      return post;
+    });
+
+    setWishDetail(newDetail);
+  };
+
+  return (
+    <StyledWishDetail>
+      {detail.map(({ id, image, title, price, like }) => (
+        <li key={id}>
+          <StyledDetailCard>
+            <div>
+              <img src={image} />
+              <HeartButton isLiked={like} toggleHeart={() => toggleHeart(id, like)} />
+            </div>
+            <span>{title}</span>
+            <div>
+              <span>₩{price.toLocaleString()}</span>
+              <span> / </span>
+              <span> 박</span>
+            </div>
+          </StyledDetailCard>
+        </li>
+      ))}
+    </StyledWishDetail>
+  );
+}
+
+export default WishDetail;
+
+const StyledWishDetail = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2.4rem;
+  padding: 0 2.2rem;
+  width: 100%;
+  li {
+    list-style: none;
+  }
+`;
+
+const StyledDetailCard = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  div:first-child {
+    position: relative;
+  }
+  div > img:first-child {
+    width: 33.1rem;
```

Comment:
지금 피그마에서 이미지 사이즈가 331px로 보여서 이렇게 한 것 같은데
이런 경우에는 화면에서 패딩만큼만 여유 있으면 되는 거라서 width: 100%으로 바꿔줘야 해!
```suggestion
    width: 100%;
```